### PR TITLE
Fixtestsfor os3

### PR DESCRIPTION
**Pull request by bhavaniravichandran**
Next set of tests to run on OKD cluster.
Will run jenkins job and update here.

### CODE REVIEW

**Review by anpanigr — APPROVED** [review]
lgtm

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java [hunk 1]
```diff
@@ -763,6 +773,8 @@ void testTwoDomainsInSameNameSpaceOnOperator() {
       HelmParams opHelmParams = installAndVerifyOperator(op2Namespace, opServiceAccount, true,
           0, op1HelmParams, domain4Namespace).getHelmParams();
       assertNotNull(opHelmParams, "Can't install operator");
+      String opExtRestRouteHost = createRouteForOKD("external-weblogic-operator-svc", op2Namespace);
```
Comment (maggiehe00) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:776]: I don't see opExtRestRouteHost is being referenced in this class. Can you explain the usage of it.

**Review by maggiehe00 — CHANGES_REQUESTED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java (hunk 1 above)
Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:776]: OK, Bhavani just explained to me on Slack.

**Comment by maggiehe00** [comment]
Bhavani, this is a big change and a lot of effort!
Can you please remove the commented out code and modify the hard coded dbNodePort. 
Also Please revisit ItParameterizedDomain.
Thanks.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiServiceMigration.java [hunk 2]
```diff
@@ -164,8 +167,15 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
...
-                         + dbNodePort + "/devpdb.k8s";
+    String dbPodIP = assertDoesNotThrow(
+        () -> getPodIP(domainNamespace, "", "oracledb"),
+        String.format("Get pod IP address failed with ApiException for oracledb in namespace %s",
+            domainNamespace));
+    logger.info("db Pod IP {0} ", dbPodIP);
+
+    //cpUrl = "jdbc:oracle:thin:@//" + K8S_NODEPORT_HOST + ":"
+    //                     + dbNodePort + "/devpdb.k8s";
+    cpUrl = "jdbc:oracle:thin:@//" + dbPodIP + ":1521" + "/devpdb.k8s";
```
[the remark this answers is not in the document]
Comment (bhavaniravichandran) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiServiceMigration.java:178]: I cannot use getNextFreePort()  here. Here the MS is trying to connect to the db listen port which is hardcoded in dbUtils.java to 1521. So, I did the same here.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java [hunk 3]
```diff
@@ -278,27 +294,32 @@ public static void initAll(@Namespaces(7) List<String> namespaces) {
     domainOnPV = createDomainOnPvUsingWdt(domainOnPVNamespace);
 
     domains.add(miiDomain);
-    domains.add(domainInImage);
-    domains.add(domainOnPV);
+    //domains.add(domainInImage);
+    //domains.add(domainOnPV);
```
[the remark this answers is not in the document]
Comment (bhavaniravichandran) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java:298]: Yes, oversight. I was tying to debug the miiusecase. Forgot to uncomment after I fixed that one. Will uncomment and run the test to make sure all the tests in that class pass.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiServiceMigration.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiServiceMigration.java:178]: OK, now I understand. You are using dbPodIP for interpod connection.

**Comment by bhavaniravichandran** [comment]
Removed the comment that prevented running the domainonpv and dii usecase parametereized test in ItParameterizedDomain.java. Result of the run in jenkins job for that test -
https://build.weblogick8s.org:8443/job/wko-okd-test/43/

**Review by maggiehe00 — APPROVED** [review]

**Pull request merged**